ParisBNF/abb/BNFabb88.xml

```diff
@@ -32,20 +32,41 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                     <msContents>
                         <summary/>
                         <msItem xml:id="ms_i1">
-                            <locus from="5r" to="172ra"/>
-                            <title type="incomplete" ref="LIT1509GadlaW">Vita and miracles of Walatta Ṗeṭros</title>
-                            <textLang mainLang="gez"/>
+                            <locus from="5r" to="178r"/>
+                                <title ref="LIT7229DossierWP"/>
+                            <msItem xml:id="ms_i1.1">
+                                <msItem xml:id="ms_i1.1.1">
+                            <locus from="5r" to="147rb"/>
+                            <title type="complete" ref="LIT1509GadlaW"/>
                             <incipit xml:lang="gez">
                                 <locus from="5ra" to="5rb"/>
                                 <hi rend="rubric">በስመ፡ እግዚአብሔር፡ ዘሃሎ፡ እምቅድመ፡</hi> ዓለም፡ <add place="margin">ወይሄሎ፡ እስከ፡ ለዓለመ፡ ዓለም፡</add> ዘአልቦ፡ ጥንት፡ ወኢተፍጻሜት፡<hi rend="rubric"> እምላክነ፡ በመለኮቱ፡ ወአቡነ፡ በኂሩቱ።</hi> ዘስብሐቲሁ፡ እምኀቤሁ፡ ወውዳሴሁ፡ እምዚአሁ፡ ዘይክል፡ ኵሎ፡ ወአልቦ፡ ዘይስአኖ፡ ለዓለመ፡ ዓለም፡ አሜን፨ አጽሕፍ፡ እንከ፡ ንስቲተ፡ አነ፡ ኃጥእ፡ ገብራ፡ <space unit="chars" quantity="7"> </space>
                                 እምዜና፡ ስደራ፡ ወእ<hi rend="rubric">ምብዙኅ፡ ገድላቲሃ፡ ወትሩፋቲሃ፡ ለእምነ፡ ቅድስት፡ ወለተ፡ ጴጥ<cb/>ሮስ፡ ሰጻፍ፡ እመ፡ አዕናቊ፡ አዕላፍ ። ። ።</hi> 
-                            </incipit> 
+                            </incipit>
+                            </msItem>
+                            <msItem xml:id="ms_i1.1.2.">
+                                <locus from="147va" to="171vb"/>
+                                <title type="complete" ref="LIT7073TammeraW"/>
+                                <incipit xml:lang="gez">
+                                    <locus from="147va" to="148rb"/>
+                                    <hi rend="rubric">በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ </hi>፩አምላክ፡ ዘለዓለም። ዓዲ፡ ናተሉ፡ ጽሒፈ፡ <hi rend="rubric">ተአምረ፡ ዘገብረት፡ እምነ፡ ቡርክት፡ ወለተ፡ ጴጥሮስ። ጸሎታ፡ ወ</hi>በረከተ፡ ረድኤታ፡ የሀሉ፡ ምስሌነ፡ ለዓለመ፡ ዓለም፡ አሜን። ስምዑ፡ ንንግርክሙ፡ ኃይለ፡ ተአምር፡ ዘኮነ፡ እምድኅረ፡ ዕረፍታ፡ ለእምነ፡ ብፅዕት፡ <hi rend="rubric">ወለተ፡ ጴጥሮስ፡</hi> ዘሰማዕነ፡ ወዘርኢነ፡ ዘይትገበር፡ በዕለተ፡ ተዝካራ፡ ለለጊዜሁ፡ እም፴ዕለት፡ እስከ፡ ፍጻሜ፡ ዓመት፡ ከመ፡ ወደረ፡ በረከተ፡ እግዚአብሔር፡ ላዕለ፡ ኵሉ፡ ዘኮነ፡ ግብረ፡ ተዝካር፡ ዘይትገበር። ወአሜሃ፡ ተጋብኡ፡ ብዙኃን፡ ሰብእ፡ እምሥራቅ፡ ወእምዕራብ፡ እምአድባር፡ ወእመካናት፡ ወእምገዳማት፡ ወእምኵሉ፡ አኅጉር፡ እለ፡ አልቦሙ፡ ኍልቊ። ቦእምኔሆሙ፡ መምህራን፡ ወቀሳውስት፡ ወዲያቆናት፡ ወመነኮሳት፡ ምስለ፡ ማዕጠንታት፡ ወመስቀላት። ወቦ፡ እምኔሆሙ፡ ነዳያን፡ ወምስኪናን፡ ወዕውራን፡ ወሐንካሳን፡ እስመ፡ ተሰምዓ፡ ዜናሃ፡ ውስተ፡ ኵሉ፡ አጽናፈ፡ ዓለም። ወወሀብዎሙ፡ ብዙኃ፡ መብልዐ፡ ወመስቴ፡ በበማዕርጊሆሙ፡ ወበበመጠሆሙ፡ እስከ፡ ጸግቡ፡ ወአትረፉ።   
+                                    ፩ተአምር፡ ዘኮነ፡ ሶቤሃ። ወይቤሉ፡ ሀለወ፡ ኅዳጥ፡ ኅቡአ፡ መጠነ፡ ፩ኅብስት፡ በዓቢይ፡ ጋን፡ ዘተረፈ፡ እምነ፡ ተዝካር። ወእምዝ፡ ተረክበ፡ እንዘ፡ ይፈልሕ፡ ወይመልዕ፡ እስከ፡ ላዕሉ፡ ወይትከዓው።  
+                                    
+                                </incipit>
+                            </msItem>
+                                <msItem xml:id="ms_i1.1.3">
```

Comment:
I do not understand what it is. This should be a description of the manuscript, it cannot have titles referring to print edition. If it is a colophon, it should be a colophon, and not an msItem. Is this an item that was copied into the manuscript from the print edition?